services/QuillLMS/engines/comprehension/comprehension.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   # Gems needed for the test environment
   s.add_development_dependency 'factory_bot_rails'
-  s.add_development_dependency "pg", '0.18.4'
+  s.add_development_dependency "pg", '~> 0.15'
```

Comment:
Why the downgrade?

Reply:
Great question. I didn't mean to commit this - I was having local issues with this version for some reason and changed it to account for that. I'm gonna take this out of the PR.